boards/arduino-nano-33-iot/Makefile

```
@@ -0,0 +1,8 @@
+MODULE = board
+
+ifneq (,$(filter boards_common_samd21-arduino-bootloader,$(USEMODULE)))
+  # Add the samd21-arduino-bootloader directory to the build
+  DIRS += $(RIOTBOARD)/common/samd21-arduino-bootloader
+endif
```

```suggestion
DIRS = $(RIOTBOARD)/common/samd21-arduino-bootloader
```

no need for the conditional